File: Weather-App/ClientApp/src/app/models/weather-forecast.ts
export interface WeatherForecast {
  id: number;
  weatherStateName: string;
  weatherStateAbbr: string;
  windDirectionCompass: string;
  created: Date;
  applicableDate: Date;
  minTemp: number;
  maxTemp: number;
  theTemp: number;
  theTempF: number;
  windSpeed: number;
  windDirection: number;
  airPressure: number;
  humidity: number;
  visibility: number;
  predictability: number;
  imgUrl: string;
  locationName: string;
}

File: Weather-App/Models/WeatherForecast.cs
using System;

namespace Weather_App.Models
{
    public class WeatherForecast
    {
        public long Id { get; set; }
        public string WeatherStateName { get; set; }
        public string WeatherStateAbbr { get; set; }
        public string WindDirectionCompass { get; set; }
        public DateTime Created { get; set; }
        public DateTime ApplicableDate { get; set; }
        public double MinTemp { get; set; }
        public double MaxTemp { get; set; }
        public double TheTemp { get; set; }
        public double TheTempF => 32 + (int) (TheTemp / 0.5556);
        public double WindSpeed { get; set; }
        public double WindDirection { get; set; }
        public double AirPressure { get; set; }
        public int Humidity { get; set; }
        public double Visibility { get; set; }
        public int Predictability { get; set; }
        public string LocationName { get; set; }
        public string ImgUrl => $"https://www.metaweather.com/static/img/weather/{WeatherStateAbbr}.svg";
    }
}